docs/releases/5.1.md

```diff
@@ -205,14 +205,32 @@ permission.save()
 
 During the deprecation period, the `permission_type` field will still be available on the `GroupPagePermission` model and is used to automatically populate empty `permission` field as part of a system check. The `permission_type` field will be removed in Wagtail 6.0.
 
-
 ### The default ordering of Group Editing Permissions models has changed
 
 The ordering for "Object permissions" and "Other permissions" now follows a predictable order equivalent do Django's default `Model` ordering.
 This will be different to the previous ordering which never intentionally implemented.
 
 This default ordering is now `["content_type__app_label", "content_type__model", "codename"]`, which can now be customised [](customising_group_views_permissions_order).
 
+### Shared include `wagtailadmin/shared/last_updated.html` is no longer available
+
+The undocumented shared include `wagtailadmin/shared/last_updated.html` is no longer available as it used the legacy Bootstrap tooltips and was not accessible. If you need to achieve a similar output, an element that shows a simple date with a tooltip for the full date, use the `human_readable_date` template tag instead.
+
+**Before**
+
+```html+django
+{% include "wagtailadmin/shared/last_updated.html" with last_updated=my_model.timestamp %}
+```
+
+**After**
```

Comment:
Note using formatted text like this as pseudo-headings is bad for screen reader users. Visually the text will look like a heading but won’t be navigable as one.

Reply:
Yeah. In the last release I did nested headings but then the heading side bar was full of before/after. 

Should I update the 5.1 release notes anyway? Maybe avoid the full depth TOC?

Reply:
all good, you did this thanks. I have updated my other PRs accordingly